src/main/java/frc/robot/Constants.java

```diff
@@ -72,7 +72,12 @@ public static class TurretConstants {
     public static final double KMAXOUTPUT = 1;
     public static final double KMINOUTPUT = -1;
     public static final int MAX_TURRET_ANGLE_DEGREES = 110;
-    public static final double CONVERSION_FACTOR = 0.25; // for the revbot prototype turret
+    public static final double GEAR_RATIO =
+        6.8333; // (24:164)  0.25 for the revbot prototype turret
+
```

Comment:
The way CONVERSION_RATIO was used here was actually the inverse of the gear ratio. So changing it to GEAR_RATIO requires inverting all the math where it is used